charts/recipes-backend/values.yaml

```
@@ -0,0 +1,25 @@
+java:
+  applicationPort: 4550
+  aadIdentityName: cnp
+  readinessPath: '/health/readiness'
+  # Don't modify below here
+  image: 'hmctspublic.azurecr.io/recipes/backend:latest'
+  environment:
+    POSTGRES_SSL_MODE: require
+  autoscaling:
+    enabled: false
+    enabledForDevMode: true
+    maxReplicas: 5
+    cpu:
+      enabled: false
+  keyVaults:
+    "plumsi":
```

```suggestion
    "plumsi":
```
Similar reference to plum specific infra?

---

same as above.